dashboard/config/environments/development.rb

```diff
@@ -62,4 +62,7 @@
 
   # see stack traces around sql queries in the log
   # ActiveRecordQueryTrace.enabled = true
+
+  # act like a levelbuilder. Set to true if you want to be able to create levels or test levelbuilder functionality
+  config.levelbuilder_mode = false
```

Comment:
Would it be easy to snag a locals.yml config param from this context? Might be a good place to read in a locals.yml config var (e.g., local_levelbuilding: true)